src/pymatgen/io/aims/parsers.py

```diff
@@ -755,6 +762,40 @@ def _parse_hirshfeld(
             "hirshfeld_dipole": hirshfeld_dipole,
         }
 
+    def _parse_mulliken(
+        self,
+    ) -> None:
+        """Parse the Mulliken charges and spins."""
+        line_start = self.reverse_search_for(["Performing Mulliken charge analysis"])
+        if line_start == LINE_NOT_FOUND:
+            self._cache.update(
+                {
+                    "mulliken_charges": None,
+                    "mulliken_spins": None,
```

Comment:
```suggestion
                    update(mulliken_charges=None, mulliken_spins=None)
```

Reply:
sorry, should have mentioned i didn't write this suggestion carefully and didn't bother with formatting either

Reply:
No worries I figured it would cause linter errors, I just did not want to have to go back and forth for all of the changes.